Changelog.md

```diff
@@ -2,6 +2,89 @@
 
+### Gazebo Physics 6.0.0 (2022-09-21)
+
+1. Improved documentation and instructions
+    * [Pull request #421](https://github.com/gazebosim/gz-physics/pull/421)
+    * [Pull request #419](https://github.com/gazebosim/gz-physics/pull/419)
+    * [Pull request #422](https://github.com/gazebosim/gz-physics/pull/422)
+    * [Pull request #417](https://github.com/gazebosim/gz-physics/pull/417)
+    * [Pull request #416](https://github.com/gazebosim/gz-physics/pull/416)
+    * [Pull request #316](https://github.com/gazebosim/gz-physics/pull/316)
+    * [Pull request #312](https://github.com/gazebosim/gz-physics/pull/312)
+    * [Pull request #401](https://github.com/gazebosim/gz-physics/pull/401)
+
+1. Added common tests
+    * [Pull request #414](https://github.com/gazebosim/gz-physics/pull/414)
+    * [Pull request #412](https://github.com/gazebosim/gz-physics/pull/412)
+    * [Pull request #411](https://github.com/gazebosim/gz-physics/pull/411)
+    * [Pull request #410](https://github.com/gazebosim/gz-physics/pull/410)
+    * [Pull request #409](https://github.com/gazebosim/gz-physics/pull/409)
+    * [Pull request #408](https://github.com/gazebosim/gz-physics/pull/408)
+    * [Pull request #407](https://github.com/gazebosim/gz-physics/pull/407)
+    * [Pull request #364](https://github.com/gazebosim/gz-physics/pull/364)
+    * [Pull request #362](https://github.com/gazebosim/gz-physics/pull/362)
+    * [Pull request #360](https://github.com/gazebosim/gz-physics/pull/360)
+    * [Pull request #363](https://github.com/gazebosim/gz-physics/pull/363)
+    * [Pull request #357](https://github.com/gazebosim/gz-physics/pull/357)
+
+1. dartsim: fix handling inertia matrix pose rotation
+    * [Pull request #351](https://github.com/gazebosim/gz-physics/pull/351)
+
+1. ABI tweaks to help bullet-featherstone
+    * [Pull request #395](https://github.com/gazebosim/gz-physics/pull/395)
+
+1. Add code coverage ignore file
+    * [Pull request #388](https://github.com/gazebosim/gz-physics/pull/388)
+
+1. Fix bullet classic entity management
+    * [Pull request #393](https://github.com/gazebosim/gz-physics/pull/393)
+
+1. Ignition to Gazebo transition
+    * [Pull request #378](https://github.com/gazebosim/gz-physics/pull/378)
+    * [Pull request #368](https://github.com/gazebosim/gz-physics/pull/368)
+    * [Pull request #356](https://github.com/gazebosim/gz-physics/pull/356)
+    * [Pull request #355](https://github.com/gazebosim/gz-physics/pull/355)
+    * [Pull request #358](https://github.com/gazebosim/gz-physics/pull/358)
+    * [Pull request #353](https://github.com/gazebosim/gz-physics/pull/353)
+    * [Pull request #348](https://github.com/gazebosim/gz-physics/pull/348)
+    * [Pull request #346](https://github.com/gazebosim/gz-physics/pull/346)
+    * [Pull request #390](https://github.com/gazebosim/gz-physics/pull/390)
+
+1. Partial source and cmake migrations
+    * [Pull request #381](https://github.com/gazebosim/gz-physics/pull/381)
```

Comment:
this is part of `Ignition to Gazebo transition`

Reply:
Done https://github.com/gazebosim/gz-physics/pull/427/commits/07e1c23094697ec22b7c1437cc87ef9c165e7cb9